src/applications/static-pages/sharable-links/sharableLink.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+const copyToUsersClipBoard = dataEntityId => {
+  const input = document.createElement('input');
+
+  const copyUrl = window.location.href.replace(window.location.hash, '');
+  input.setAttribute('value', `${copyUrl}#${dataEntityId}`);
+  document.body.appendChild(input);
+  input.select();
+  document.execCommand('copy');
+  document.body.removeChild(input);
+};
+const ShareIconClickFeedback = styled.span`
+  position: ${props => (props.leftAligned ? 'absolute' : 'relative')};
+  margin-top: ${props => (props.leftAligned ? '2px' : '')};
+  left: ${props =>
+    props.leftAligned && props.leftPx ? `${props.leftPx}px` : ''};
+  background-color: black;
+  color: white;
+  font-family: 'Source Sans Pro';
+  font-weight: 400;
+  font-size: 12px;
+  height: 26px;
+  padding: 4px;
+  border: 1px solid;
+  bottom: ${props => (props.leftAligned ? '' : '1px')};
+`;
+
+const ShareIcon = styled.i`
+  border-radius: 5px;
+  font-size: 16px;
+  height: 26px;
+  width: 26px;
+  padding: 4px;
+  border: 1px solid;
+  color: #0071bb;
```

Comment:
Do we need this specific color, or can we use one of the standard design system [colors](https://design.va.gov/design/color-palette)?

Reply:
I used the primary color from here. I will convert it to a variable in a follow on commit to this PR:
https://design.va.gov/utilities/color